content/post/2018-06-05_badcomkillsgoodanalytics.md

```diff
@@ -0,0 +1,88 @@
+---
+author: Randy Clinton
+title: "Bad communication kills good analytics"
+cover: MISSING
+publishDate: UNKNOWN
+tags:
+ - teams
+ - communication
+ - presentation
+---
+
+Poor communication within an Analytics team and between that team and the company, leave highly skilled analytics employees solving the wrong questions, lacking support for big ideas and leaving the company unsatisfied with their work. Here, I'll discuss ways a team can improve by focusing on constructive conversions.
+<!--more-->
+
+Analytics team exist because companies wants better conversations. Technical tools allow us ways to find information, but at the end we need to talk about it. Without an analytics team, a company will discuss business decisions like marketing spending, product offerings, and ways to encourage customers repurchase. A productive analytics team grounds these conversations in truth and testing.
```

Comment:
s/wants/want

Reply:
This sentence feels like it's missing something:
> Without an analytics team, a company will discuss business decisions like marketing spending, product offerings, and ways to encourage customers repurchase.

Is this a good or a bad thing? It sounds like a good thing but I think you're trying to set up the problem for which the analytics team has a solution!

Reply:
agree. new graph:

>Analytics team exist because companies want better conversations. Technical tools allow us ways to find information, but at the end we need to talk about it. A productive analytics team grounds these conversations in truth and testing. A discuss about marketing ad spend is better when the Analytics team brings in an understandable attribution model. A decision about product offerings is better when the Analytics team shares market research. Strategies to increase re-purchases are improved when the Analytics team has customer behavior analysis.